app/views/pages/dashboard_blocks/_cars.html.haml

```diff
@@ -1,3 +1,10 @@
-%ul.list-group
-  - data[:cars].each do |car|
-    %li.list-group-item.bg-secondary #{car.name}
+%h4 #{ t('.last_added')}
+
+%table.table.table-sm
+  %thead
+    %tr
+      %th{scope: 'col'} Name
```

Comment:
Please use everywhere 'translation'. Look at Kamil's PR https://github.com/workshop-team/motohelper/pull/57.
http://guides.rubyonrails.org/i18n.html#translations-for-active-record-models